kumascript/macros/SpecData.json

```diff
@@ -146,7 +146,7 @@
   },
   "CSP 1.1": {
     "name": "Content Security Policy Level 2",
-    "url": "https://w3c.github.io/webappsec-csp/2/",
+    "url": "https://www.w3.org/TR/CSP2/",
```

Comment:
I suspect you accidentally started on an outdated `main` or something.

Reply:
@peterbe Thanks. While that may seem likely I believe it to have been a github issue - since I went though github interface on main branch and did not touch this file.